tests/FSharp.Compiler.ComponentTests/Conformance/BasicGrammarElements/AccessibilityAnnotations/PermittedLocations/PermittedLocations.fs

```diff
@@ -27,12 +27,12 @@ module AccessibilityAnnotations_PermittedLocations =
         |> verifyCompile
         |> shouldFail
         |> withDiagnostics [
-            (Error 561, Line 18, Col 5, Line 18, Col 62, "Accessibility modifiers are not allowed on this member. Abstract slots always have the same visibility as the enclosing type.")
-            (Error 561, Line 19, Col 5, Line 19, Col 62, "Accessibility modifiers are not allowed on this member. Abstract slots always have the same visibility as the enclosing type.")
-            (Error 561, Line 20, Col 5, Line 20, Col 62, "Accessibility modifiers are not allowed on this member. Abstract slots always have the same visibility as the enclosing type.")
-            (Error 10, Line 21, Col 14, Line 21, Col 20, "Unexpected keyword 'public' in member definition. Expected identifier, '(', '(*)' or other token.")
-            (Error 561, Line 21, Col 14, Line 22, Col 62, "Accessibility modifiers are not allowed on this member. Abstract slots always have the same visibility as the enclosing type.")
-            (Error 10, Line 23, Col 14, Line 23, Col 22, "Unexpected keyword 'internal' in member definition. Expected identifier, '(', '(*)' or other token.")
+            (Error 531, Line 18, Col 5, Line 18, Col 11, "Accessibility modifiers should come immediately prior to the identifier naming a construct")
```

Comment:
Does this really make things better here?

This would now pop up for the line:
```fsharp
type private IDoStuff =
    public abstract SomeStuffa1 : int -> int -> (int * int)
``` 

The errors tells users to switch `public` and `abstract` but that's not the main problem here and the switch will then lead to the next error, the one that used to be there before.

I'd say, we should keep this behavior.

Reply:
> I'd say, we should keep this behavior.

Can we show both FS0531 and FS0561 to fix this?

Reply:
Yeah, this looks reasonable.